In Dark mode,  in the reference section the syntax  text color should be in black

### Most appropriate sections of the p5.js website?

Reference

### What is your operating system?

Windows

### Web browser and version

_No response_

### Actual Behavior

In the reference section of circle the syntax text color should also be change. Currently is it in black color
![Image](https://github.com/user-attachments/assets/ab08c156-ee89-4ee1-aa18-d48548501395)

![Image](https://github.com/user-attachments/assets/f01b35a3-f398-4719-bdb8-73b24d3594ee)

### Expected Behavior

It should white in text color 

### Steps to reproduce

_No response_

### Would you like to work on the issue?

Please assign me this issue. I would love to work on this issue

p5.js-website (main)$ npm run dev

> p5.js-website@0.0.1 dev
> astro dev

12:39:45 am [vite] Error when evaluating SSR module C:\Users\Piyush\gsoc\p5.js-website\astro.config.mjs: failed to import "astro-compress"
|- Error:
Something went wrong installing the "sharp" module

Cannot find module '../build/Release/sharp-win32-x64.node'
Require stack:
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\sharp.js
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\constructor.js
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\index.js

Possible solutions:
- Install with verbose logging and look for errors: "npm install --ignore-scripts=false --foreground-scripts --verbose sharp"
- Install for the current win32-x64 runtime: "npm install --platform=win32 --arch=x64 sharp"
- Consult the installation documentation: https://sharp.pixelplumbing.com/install   
    at Object.<anonymous> (C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\sharp.js:37:9)
    at Module._compile (node:internal/modules/cjs/loader:1739:14)
    at Object..js (node:internal/modules/cjs/loader:1904:10)
    at Module.load (node:internal/modules/cjs/loader:1473:32)
    at Function._load (node:internal/modules/cjs/loader:1285:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:234:24)
    at Module.require (node:internal/modules/cjs/loader:1495:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\constructor.js:11:1)

[astro] Unable to load your Astro config


Something went wrong installing the "sharp" module

Cannot find module '../build/Release/sharp-win32-x64.node'
Require stack:
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\sharp.js
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\constructor.js
- C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\index.js

Possible solutions:
- Install with verbose logging and look for errors: "npm install --ignore-scripts=false --foreground-scripts --verbose sharp"
- Install for the current win32-x64 runtime: "npm install --platform=win32 --arch=x64 sharp"
- Consult the installation documentation: https://sharp.pixelplumbing.com/install   
  Hint:
    Browser APIs are not available on the server.

    If the code is in a framework component, try to access these objects after rendering using lifecycle methods or use a `client:only` directive to make the component exclusively run on the client.

    See https://docs.astro.build/en/guides/troubleshooting/#document-or-window-is-not-defined for more information.

  Stack trace:
    at Object.<anonymous> (C:\Users\Piyush\gsoc\p5.js-website\node_modules\astro-compress\node_modules\sharp\lib\sharp.js:37:9)
    at Object..js (node:internal/modules/cjs/loader:1904:10)
    at Function._load (node:internal/modules/cjs/loader:1285:12)
    at wrapModuleLoad (node:internal/modules/cjs/loader:234:24)
    at require (node:internal/modules/helpers:135:16)



I see this error while running command npm run dev

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
1 │ @import "variables.scss";
  │         ^^^^^^^^^^^^^^^^
  ╵
    styles\global.scss 1:9                     @import
    src\components\Nav\styles.module.scss 2:9  root stylesheet

Deprecation Warning [legacy-js-api]: The legacy JS API is deprecated and will be removed in Dart Sass 2.0.0.

More info: https://sass-lang.com/d/legacy-js-api

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
1 │ @import "./variables.scss";
  │         ^^^^^^^^^^^^^^^^^^
  ╵
    styles\base.scss 1:9  root stylesheet

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
2 │ @import "./global.scss";
  │         ^^^^^^^^^^^^^^^
  ╵
    styles\base.scss 2:9  root stylesheet

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
3 │ @import "./code-editor.scss";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
    styles\base.scss 3:9  root stylesheet

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
4 │ @import "./markdown.scss";
  │         ^^^^^^^^^^^^^^^^^
  ╵
    styles\base.scss 4:9  root stylesheet

Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
1 │ @import "variables.scss";
  │         ^^^^^^^^^^^^^^^^
  ╵
    styles\global.scss 1:9  @import
    styles\base.scss 2:9    root stylesheet

Deprecation Warning [mixed-decls]: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming        
version. To keep the existing behavior, move the declaration above the nested       
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

   ╷
29 │ ┌     @media (min-width: $breakpoint-tablet) {
30 │ │       min-width: 220px;
31 │ │     }
   │ └─── nested rule
32 │       color: var(--sidebar-type-color);
   │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
   ╵
    styles\markdown.scss 32:5  @import
    styles\base.scss 4:9       root stylesheet

Warning: 1 repetitive deprecation warnings omitted.

Deprecation Warning [legacy-js-api]: The legacy JS API is deprecated and will be removed in Dart Sass 2.0.0.

More info: https://sass-lang.com/d/legacy-js-api

01:15:48 [ERROR] Cannot find module 'C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\useCodeMirror' imported from C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\index.js
  Stack trace:
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:640:25)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:276:38) 
01:15:48 [ERROR] Cannot find module 'C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\useCodeMirror' imported from C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\index.js
  Stack trace:
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:640:25)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:276:38)

Cannot find module 'C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\useCodeMirror' imported from C:\Users\Piyush\gsoc\p5.js-website\node_modules\@uiw\react-codemirror\esm\index.js   

Now getting this error . I have remove the node_modules 2 to 3 times but still getting the same error and I have install @uiw/react-codemirror/esm locally many times but still not working @davepagurek  @limzykenneth

--- a/src/components/MethodSignature/index.astro
+++ b/src/components/MethodSignature/index.astro
@@ -32,7 +32,7 @@ const signature = `${name}(${formattedParams})`;
 {
   formattedParams.map((params) => (
     <CodeContainer>
-      <span>{`${name}(${params})`}</span>
+      <span class="text-black">{`${name}(${params})`}</span>
       <div class="absolute top-xs right-xs">
         <CopyCodeButton client:load textToCopy={signature} />
       </div>

